src/networking/http-client.ts

```diff
@@ -103,8 +105,11 @@ function getHeaders(
     "X-Version": VERSION,
     "X-Is-Sandbox": `${isSandboxApiKey(apiKey)}`,
   };
-  if (headers != null) {
+  if (headers) {
     all_headers = { ...all_headers, ...headers };
   }
+  if (additionalHeaders) {
+    all_headers = { ...additionalHeaders, ...all_headers };
```

Comment:
Could we change this for consistency?
```suggestion
    all_headers = { ...all_headers, ...additionalHeaders };
```